src/uwtools/api/config.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=unused-import
+
+from typing import Optional
+
+from uwtools.config.atparse_to_jinja2 import convert as _convert_atparse_to_jinja2
+from uwtools.config.tools import compare_configs as _compare
+from uwtools.config.tools import realize_config as _realize
+from uwtools.config.validator import validate_yaml as _validate
```

Comment:
Care has been taken to import code from other modules with private local names. Note that, for example, what this module makes public and what is kept private:
```
>>> import uwtools.api.config
>>> dir(uwtools.api.config)
['DefinitePath', 'FORMAT', 'Optional', 'OptionalPath', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__spec__', '_compare', '_convert_atparse_to_jinja2', '_realize', '_validate', 'compare', 'realize', 'translate', 'validate']
```
Only the API functions `compare()`, `realize()`, `translate()`, and `validate()` are public -- everything else is private. For example, users should not call `uwtools.config.atparse_to_jinja2.convert_atparse_to_jinja2()` directly, so it is imported privately in this module to avoid exposing it.

Reply:
All of the above boils down to a simple recipe for API users:

1. Only use the `uwtools.api` package.
2. Only use the public members of that package.